### PR TITLE
Add elixir events

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: 'Training Infrastructure Feedback on the ELIXIR Belgium workshop “DDA and DIA proteomic analysis in Galaxy” '
+tags: [training, TIaaS]
+supporters:
+- galaxy-europe
+- TIaaS
+author: Matthias Fahrner & Melanie Föll
+---
+
+On December 3 and 4, we ran a 2-day Galaxy Proteomics data analysis workshop organized by ELIXIR Belgium. Therefore, most of the 20 participants were from Belgium, but others were from all around Europe, the US and Australia.
```
Comment (bgruening) [inline _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md:11]: 3rd and 4th?

File: _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+We held lectures and demos of the respective GTN material in the morning until noon while answering questions and discussing details of the involved analysis steps. In the afternoon, the participants had time to go through the training at their own pace using the GTN material, and asking questions in a shared google doc as well as live in the zoom meeting. Due to the designated computational resources, the waiting time was minimized and we were able to track the progress of our participants in going through the training. Many thanks to the Galaxy Training Infrastructure as a Service, without which this course would not have been such a success!
```
Comment (bgruening) [inline _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md:15]: in a zoom meeting?

Comment (bgruening) [reply _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md:15]: track the progress with the TIaaS dashboard - link?

**Review by bgruening — COMMENTED** [review]
I will also ask if we can link the training material.

File: _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2020-12-08-tiaas_feedback_Melanie-Matthias.md:11]: I think that this is correct as it is

**Review by bgruening — APPROVED** [review]

**Pull request merged**